src/pull.go

```diff
@@ -82,6 +88,72 @@ func (g *Commands) Pull(byId bool) error {
 	return g.playPullChanges(nonConflicts, g.opts.Exports, opMap)
 }
 
+func autoRenameClashes(g *Commands, clashes []*Change) error {
+	type rename struct {
+		name string
+		change *Change
+	}
+	clashesMap := map[string][]rename{}
+
+	for _, clash := range clashes {
+		group := clashesMap[clash.Path]
+		if clash.Src == nil { // Is that even possible? In what situation?
+			continue
+		}
+		clashesMap[clash.Path] = append(group, rename{"", clash})
+	}
+
+	for path, group := range clashesMap {
+		ext := filepath.Ext(path)
+		name := strings.TrimSuffix(path, ext)
+		n := 0
+
+		for i, r := range group[1:] {  // we can leave first item with original name
+			var newName string
+			for {
+				newName = fmt.Sprintf("%v_%d%v", name, n, ext)
+				n++
```

Comment:
Sure, no hurry. When you're done reviewing the stuff and I'm done fixing it, I believe it would be cleaner to stash all my commits to one and add proper commit message etc. What do you think?